### ISSUE TITLE
fatal: No path specified. See 'man git-pull' for valid url syntax

### ISSUE THREAD

**Issue by tm4g1x** [angular-schule/angular-cli-ghpages#98]
Hi,

I am having a bit of trouble getting the deployment done via a bamboo CI/CD  job... I followed the explanation for GH actions and until the deployment everything works fine. When I run the deployment from my local machine, everything is deployed as wanted. 

Personal access token etc. is also set up and added.

![image](https://user-images.githubusercontent.com/24810353/79982943-5513fe80-84a7-11ea-9654-167e43c0cb1c.png)

Any idea how to fix this?

Cheers

**Comment by JohannesHoppe** [comment]
Please show us the exact command you executed.

**Comment by tm4g1x** [comment]
Cannot share exact command information... adjusted the command so repo, name and email address is replaced in it. It is a private repo on an on prem github (Actions are not available there, that's why I try it out with bamboo)

it is run in an npm task in bamboo, as env. variable GH_TOKEN is added which grants right to write into my repos.
npm task:
run ng -- deploy --no-silent  --base-href=/repo/ --name="name" --email=test@test.de

Git hub repo is checked out before, which works... Build is successful, only deploy fails

**Comment by JohannesHoppe** [comment]
In this case you have to provide --repo, --name and --email, too.

**Comment by tm4g1x** [comment]
actually I tried that but it did not work. I found a solution now...

In a shell script the git config needs to be adapted to contain the correct user name and email. But this is not sufficient yet. It was further required to remove the remote origin and add it back in with "https://user:personalaccesstoken@github.com/repo.git"

May this steps are bamboo specific, but maybe someone else runs into it as well.

Cheers and thanks for the cli enhancement
Tom
